bigtable/integration_test.go

```diff
@@ -1450,6 +1477,186 @@ func TestIntegration_AdminCreateInstance(t *testing.T) {
 	if cInfo.ServeNodes != 5 {
 		t.Fatalf("NumNodes: %v, want: %v", cInfo.ServeNodes, 5)
 	}
+
+	if cInfo.KMSKeyName != "" {
+		t.Fatalf("KMSKeyName: %v, want: %v", cInfo.KMSKeyName, "")
+	}
+}
+
+func TestIntegration_AdminEncryptionInfo(t *testing.T) {
+	if instanceToCreate == "" {
+		t.Skip("instanceToCreate not set, skipping instance creation testing")
+	}
+
+	testEnv, err := NewIntegrationEnv()
+	if err != nil {
+		t.Fatalf("IntegrationEnv: %v", err)
+	}
+	defer testEnv.Close()
+
+	if !testEnv.Config().UseProd {
+		t.Skip("emulator doesn't support instance creation")
+	}
+
+	timeout := 5 * time.Minute
+	ctx, _ := context.WithTimeout(context.Background(), timeout)
+
+	iAdminClient, err := testEnv.NewInstanceAdminClient()
+	if err != nil {
+		t.Fatalf("NewInstanceAdminClient: %v", err)
+	}
+	defer iAdminClient.Close()
+
+	adminClient, err := testEnv.NewAdminClient()
+	if err != nil {
+		t.Fatalf("NewAdminClient: %v", err)
+	}
+	defer adminClient.Close()
+
+	table := testEnv.Config().Table
+	clusterID := testEnv.Config().Cluster
+
+	// TODO: It may be good to automate, or allow this to be passed in ;)
+	project := "firestore-harvard-library"
+	loc := "us-central1" // NOTE: cannot use "global"
+	ring := "crwilcox-cmek-test-key2"
```

Comment:
note: today this is hardcoded to a key. I can make this a configuration variable at least and have CI use it. Would be ideal to automate creation, or use an available cmek key on the project.

Reply:
Not sure how comparable this is to storage CMEK, but I think for that we require the person running the tests to create a keyring themself and set and env var to it.

Reply:
I think this is going to be the case here also. That said, if we already have that set for storage, I can likely use the same key.

Reply:
used key we have for other cmek stuffs.